src/internal/operations/restore.go

```diff
@@ -138,11 +139,29 @@ func (op *RestoreOperation) Run(ctx context.Context) (restoreDetails *details.De
 
 	ctx = count.Embed(ctx, op.Counter)
 
+	var cats []string
+
+	allCats, err := op.Selectors.AllPathCategories()
+	if err != nil {
+		// No need to exit over this, we'll just be missing a bit of info in the
+		// log.
+		logger.CtxErr(ctx, err).Info("getting categories for restore")
+	} else {
+		for _, cat := range allCats {
+			cats = append(cats, cat.HumanString())
+		}
+	}
```

Comment:
Since this is the same code in all three locations, and the AllPathCategories was created specifically for this, how about either 1) packing the humanString and sort into AllPathCategories() or 2) adding an AllHumanCategories() func that calls AllPathCategories() and runs the transform/sort in that func?